packages/dx-react-scheduler-demos/src/demo-sources/scheduler-featured-integrated-data-shaping/material-ui/demo.tsx

```diff
@@ -0,0 +1,283 @@
+    endDate: new Date(2018, 6, 4, 11, 0),
+    priority: 1,
+    location: 'Room 1',
+  }, {
+    title: 'Final Budget Review',
+    startDate: new Date(2018, 6, 6, 12, 0),
+    endDate: new Date(2018, 6, 6, 13, 35),
+    priority: 3,
+    location: 'Room 1',
+  }, {
+    title: 'New Brochures',
+    startDate: new Date(2018, 6, 6, 14, 30),
+    endDate: new Date(2018, 6, 6, 15, 45),
+    priority: 3,
+    location: 'Room 3',
+  }, {
+    title: 'Install New Database',
+    startDate: new Date(2018, 6, 10, 9, 45),
+    endDate: new Date(2018, 6, 10, 11, 15),
+    priority: 2,
+    location: 'Room 2',
+  }, {
+    title: 'Approve New Online Marketing Strategy',
+    startDate: new Date(2018, 6, 12, 12, 0),
+    endDate: new Date(2018, 6, 12, 14, 0),
+    priority: 1,
+    location: 'Room 2',
+  }, {
+    title: 'Upgrade Personal Computers',
+    startDate: new Date(2018, 6, 16, 15, 15),
+    endDate: new Date(2018, 6, 16, 16, 30),
+    priority: 2,
+    location: 'Room 3',
+  }, {
+    title: 'Customer Workshop',
+    startDate: new Date(2018, 6, 18, 11, 0),
+    endDate: new Date(2018, 6, 18, 12, 0),
+    priority: 3,
+    location: 'Room 1',
+  }, {
+    title: 'Prepare 2015 Marketing Plan',
+    startDate: new Date(2018, 6, 20, 11, 0),
+    endDate: new Date(2018, 6, 20, 13, 30),
+    priority: 1,
+    location: 'Room 3',
+  },
+  {
+    title: 'New Brochures',
+    startDate: new Date(2018, 6, 23, 14, 30),
+    endDate: new Date(2018, 6, 23, 15, 45),
+    priority: 2,
+    location: 'Room 3',
+  }, {
+    title: 'Install New Database',
+    startDate: new Date(2018, 6, 23, 9, 45),
+    endDate: new Date(2018, 6, 23, 11, 15),
+    priority: 3,
+    location: 'Room 2',
+  }, {
+    title: 'Approve New Online Marketing Strategy',
+    startDate: new Date(2018, 6, 26, 12, 0),
+    endDate: new Date(2018, 6, 26, 14, 0),
+    priority: 1,
+    location: 'Room 1',
+  }, {
+    title: 'Upgrade Personal Computers',
+    startDate: new Date(2018, 6, 31, 15, 15),
+    endDate: new Date(2018, 6, 31, 16, 30),
+    priority: 2,
+    location: 'Room 3',
+  }, {
+    title: 'Install New Database',
+    startDate: new Date(2018, 6, 31, 9, 45),
+    endDate: new Date(2018, 6, 31, 11, 15),
+    priority: 3,
+    location: 'Room 2',
+  },
+];
+
+const styles = ({ palette }: Theme) => createStyles({
+  appointment: {
+    borderRadius: 0,
+  },
+  firstRoomAppointment: {
+    backgroundColor: indigo[300],
+    '&:hover': {
+      backgroundColor: indigo[400],
+    },
+  },
+  secondRoomAppointment: {
+    backgroundColor: blue[300],
+    '&:hover': {
+      backgroundColor: blue[400],
+    },
+  },
+  thirdRoomAppointment: {
+    backgroundColor: teal[300],
+    '&:hover': {
+      backgroundColor: teal[400],
+    },
+  },
+  highPriorityAppointment: {
+    borderLeft: `4px solid ${teal[500]}`,
+  },
+  middlePriorityAppointment: {
+    borderLeft: `4px solid ${blue[500]}`,
+  },
+  lowPriorityAppointment: {
+    borderLeft: `4px solid ${indigo[500]}`,
```

Comment:
To remove this display bug set please`borderBottom: 0`.
![image](https://user-images.githubusercontent.com/15093892/67005691-e0bd6480-f0eb-11e9-9b28-1307b14dd81c.png)